### ISSUE TITLE
Waiting for a response..." times out

### ISSUE THREAD

**Issue by BeachGuy007** [necarlson97/codex-ai-sublime#8]
Waiting for response then times out.

**Comment by necarlson97** [comment]
Yes - sorry, the openai api has undergone significant changes since the last update of this package. I will try to attend to this when I can find time - but if anyone else wants to try their hand at a PR to update to the current api, it 'should' be relatively straightforward.
Thank you for the alert!
